queries/php/highlights.scm

```diff
@@ -142,7 +142,6 @@
 (null) @constant.builtin
 (integer) @number
 (float) @float
-(comment) @comment @spell
```

Comment:
This I don't agree with. It can be disabled by the user through `(comment) @nospell`, but unsure what would the effect be with other `@spell` of phpdoc tho?

Reply:
I realize that disabling spell checking of comments at the php level can be controversial. IMHO it doesn't do its job well (it checks variable names, types etc.). At the phpdoc level we can do it more precisely.

Reply:
yep, it'll be controversial. Personally I don't use comment-related parsers (comment, phpdoc, ...) so some spellchecks that doesn't rely on the parsers will be more preferrable. 

Up to the php maintainers to decide. 
cc @tk-shirasaka

Reply:
No way to recognize if the comment is phpdoc or not (and enable spell on non-phpdoc comments?)?

Reply:
> No way to recognize if the comment is phpdoc or not (and enable spell on non-phpdoc comments)?

You can keep `(comment) @comment @spell` in `php`, while having a `(text) @nospell` at the top of the phpdoc file to counter that `@spell`, then have `(description (text) @spell)` to re-enable it for that section only, I believe

Reply:
> You can keep `(comment) @comment @spell` in `php`, while having a `(text) @nospell` at the top of the phpdoc file to counter that `@spell`, then have `(description (text) @spell)` to re-enable it for that section only, I believe

It does not work for me. 

At the moment my best approach that resolves the problem without disabling `@spell` at the php level for the whole comment is to put `@nospell` for particular nodes like in the second commit (c2297a6ffd4396e2dce7ff36961d914c391e50ee).

Reply:
Let's try with the best approach you have :D 

If anything, it's probably the most correct one imo